src/main/java/com/google/api/codegen/transformer/ImportTypeTransformer.java

```diff
@@ -61,4 +74,11 @@
 
     return imports;
   }
+
+  private String getFileImports(
```

Comment:
This name is misleading because it is plural, but it only returns one import. Maybe call it `getFileImportFromService`.